### PR TITLE
Add maximum resolution note to location accuracy notice

### PR DESCRIPTION
**Pull request by PockyBum522**
I had to look up what the setting was allowed to be, so I figured I'd relay that information where it may be useful.

### CODE REVIEW

**Review by brianshea2 — COMMENTED** [review]
This is great, thank you. The only thing I'd prefer to change before committing is putting this note just after the quote, since it's not actually part of the quote, and then the parentheses could be removed as well.

**Comment by PockyBum522** [comment]
Moved it to its own line, per instruction. As it is now slightly more of a standalone concept, I added a small amount of defining text. Hopefully this feels appropriate. Thanks!

File: README.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ If your node can be heard by another node already reporting to MQTT, that's it!
 #### Important update as of August, 2024
 Meshtastic has [made a change to their MQTT server](https://meshtastic.org/blog/recent-public-mqtt-broker-changes/):
 
-> Only position packets with imprecise location information [10-16 bits] will be passed to the topic, ensuring that sensitive data is not exposed.
+Only position packets with imprecise location information [10-16 bits] will be passed to the topic, ensuring that sensitive data is not exposed.
```
Comment (brianshea2) [inline README.md:25]:
```suggestion
> Only position packets with imprecise location information [10-16 bits] will be passed to the topic, ensuring that sensitive data is not exposed.
```
let's keep this a quote

**Comment by PockyBum522** [comment]
My bad. I apparently hadn't had my coffee yet and didn't realize that was markdown.

**Review by brianshea2 — APPROVED** [review]

**Pull request merged**